### PR TITLE
[ci skip] Broken links in documentation fix

### PR DESCRIPTION
**Pull request by kjellberg**
Fixed broken links in documentation :) 

Release notes from 2.2 to 4.2 also have a lot of broken links but I don't know if they should be replaced or not so I didn't touch them.

### CODE REVIEW

**Comment by maclover7** [comment]
Hi @kjellberg, can you please squash down your commits to just one?

**Comment by kjellberg** [comment]
Was already working on that... You're to fast 💃  👍  Sorry, next time I hope that I manage to do it right from the beginning 🐙

File: guides/source/security.md [hunk 1]
```diff
@@ -287,7 +287,7 @@ Another class of security vulnerabilities surrounds the use of redirection and f
...
+Whenever the user is allowed to pass (parts of) the URL for redirection, it is possibly vulnerable. The most obvious attack would be to redirect users to a fake web application which looks and feels exactly as the original one. This so-called phishing attack works by sending an unsuspicious link in an email to the users, injecting the link by XSS in the web application or putting the link into an external site. It is unsuspicious, because the link starts with the URL to the web application and the URL to the malicious site is hidden in the redirection parameter: http://www.example.com/site/redirect?to=www.attacker.com. Here is an example of a legacy action:
```
Comment (maclover7) [inline guides/source/security.md]: Sorry, I think this is just the GitHub web UI being weird, but what was the change made to this paragraph?

Comment (kjellberg) [inline guides/source/security.md]: Just a small fix
?to= www.attacker.com changed to ?to=www.attacker.com (space between ?to= and www)

Comment (maclover7) [inline guides/source/security.md]: Gotcha, thanks!

**Comment by maclover7** [comment]
:+1: Thank you for your contribution!

:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**